README.markdown

```diff
@@ -833,6 +834,29 @@ while i < attendeeList.count {
     i += 1
 }
 ```
+
+## Third Party Imports
+
+Third party libraries that have yet to annotate their Objective-C headers for nullability will be imported into swift with implicitly unwrapped optionals. These should be treated as optionals and guarded on use as any other optional parameter. For example, a third party library that gets imported might look to swift as:
+
+```swift
+public class ThirdPartyDelegate : NSObject {
+    public func onError(errorInfo: ErrorInfo!)
+}
+```
+
+An implementing subclass should implement this delegate by changing the implicitly unwrapped optional into an optional.
```

Comment:
> An implementing subclass should implement this delegate by changing the implicitly unwrapped optional into an optional.

A overriding subclass should instead treat implicitly unwrapped optional parameters as optionals.